java/code/src/com/redhat/rhn/manager/action/ActionManager.java

```diff
@@ -1950,6 +1952,41 @@ public static Action schedulePackageAction(User scheduler, List<Map<String, Long
         return action;
     }
 
+
+    private static List<Map<String, ? extends Object>> removeDuplicatedName(List<Map<String, Long>> packageMaps) {
+        Map<String, Map<String, Long>> packageMapsWithoutDuplicated = new HashMap<>();
+
+        for (Map<String, Long> map : packageMaps) {
+
+            Map<String, Long> newMap = map.entrySet().stream()
+                 .collect(Collectors.toMap(Map.Entry::getKey, e -> e.getValue()));
```

Comment:
I wonder a bit if `newMap` is somehow different from `map` ?
Both seems to be just "String" => "Long" .

Reply:
right, it  was required before cause `newMap` was String -> Object, now it can be removed. I'll do that and in case remove it

Reply:
fixed